ext/test/w3c_tracecontext_test/CMakeLists.txt

```diff
@@ -10,4 +10,8 @@ else()
     PRIVATE ${CMAKE_THREAD_LIBS_INIT} opentelemetry_trace http_client_curl
             opentelemetry_exporter_ostream_span ${CURL_LIBRARIES}
             nlohmann_json::nlohmann_json)
+  if(lohmann_json_clone)
```

Comment:
should it be `nlohmann_json_clone` ? I think CI uses external nlohmann_json package, so it won't get caught there :)

Reply:
thanks. fixed